### PR TITLE
Ab#5690 catalan translations

### PR DESCRIPTION
**Pull request by PeculiarGoat**
https://dev.azure.com/S72/SHIFT72/_workitems/edit/5690/

### CODE REVIEW

File: kibble.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "defaultLanguage": "en",
   "languages": {
     "ar": { "code": "ar_LB", "name": "عربى" },
+    "ca": { "code": "ca_ES", "name": "Català" },
```
Comment (stajs) [inline kibble.json:9]:
```suggestion
    "ca": { "code": "es_CA", "name": "Català" },
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased](https://github.com/shift72/core-template/compare/0.3.2...HEAD)
 
+### Added
+- Added Catalan (`ca_ES`) translations file.
```
Comment (stajs) [inline CHANGELOG.md:6]:
```suggestion
- Added Catalan (`es_CA`) translations file. 
```

**Review by stajs — CHANGES_REQUESTED** [review]

File: kibble.json (hunk 1 above)
Comment (PeculiarGoat) [reply kibble.json:9]: `ca_ES` is lang_REGION though...

Comment (stajs) [reply kibble.json:9]: You are right! My bad, I thought it was Spanish as spoken in Catalan - but Catalan is a language not a country/region. In which case the future convention would be just `ca`.

**Review by stajs — APPROVED** [review]

**Pull request merged**